### ISSUE TITLE
metno and Vol C1 for stations with WSI

### ISSUE THREAD

**Issue by efucile** [wmo-im/WSI#8]
There isn't any guidance on how to produce a metno with WSI and Vol C1 is not ready to register entries with WSI.
In WIS metno and Vol C1 should be replaced by functions of the GISC, but this is not the case yet.

**Comment by josusky** [comment]
Producers of meteorological data are, for quite some time, required to create/update the WIS metadata record that corresponds to the product they intend to add/change. I am sure that there are some omittions and human errors, same as there were with Vol C1. 
But the good news is that BUFR encoded data (WSI can be only in a BUFR) are much more self descriptive than TAC. In addition to station identifier (WSI and/or TSI) there are coordinates, thus, there are less reasons to consult a station catalogue on reception of a BUFR message. Consequently, if an organization publishes a metadata record that properly describes meaning of the data (messages) and its availability, but does not specify all station identifiers ... well, then such records is more valuable than another one (quite common at the moment) that meticulously lists all stations but does not tell what the data is about.
As long as the WSIs are assigned and used consistently (that is other topic), I would not stress much detailing of stations lists in metadata about observations.

**Comment by kaiwirt** [comment]
In 2017 GISCs recommended to put WSI into the remarks field of Vol C1 for the time being. However for WIS 2 it would be useful to link WIS and OSCAR somehow.

**Comment by golfvert** [comment]
@kaiwirt can you detail what you mean by "However for WIS 2 it would be useful to link WIS and OSCAR somehow"? Practically what would that mean? "Real" technical stuff or only something in the Manual or equivalent?

**Comment by kaiwirt** [comment]
I am talking about technical stuff.

In WIS Metadata we have a list of stations which are contained in the corresponding bulletin. However this is only text and there is no way to get background Information on the stations. (In fact this is Volume C)

In OSCAR we have Background Information on stations. However there is no way to discover all bulletins which contain data from this station.

What i tried to say was, that it would be good if there was a link between the two metadata catalogues such that you can get Information on the stations starting in WIS and that you can get a list of relevant bulletins by starting from a station entry in OSCAR.

**Comment by benjaminsaclier** [comment]
Hi all,

In WIS metadata I think there is a recommendation to link the metadata for stations  which have been assigned a WIGOS Station Identifier with OSCAR under the descriptive keyword 'place'. We used it for french stations metadata (for example : 
http://wispi.meteo.fr/openwis-user-portal/srv/en/iso19139.xml?id=438896). 

_<gmd:descriptiveKeywords><gmd:MD_Keywords id="place"><gmd:keyword><gco:CharacterString>0-20000-0-07145</gco:CharacterString></gmd:keyword><gmd:keyword><gco:CharacterString>TRAPPES [ http://data.wmo.int/wigosid=0-20000-0-07145 ]</gco:CharacterString></gmd:keyword><gmd:keyword><gco:CharacterString>0-20000-0-07149</gco:CharacterString></gmd:keyword><gmd:keyword><gco:CharacterString>ORLY [ http://data.wmo.int/wigosid=0-20000-0-07149 ]</gco:CharacterString></gmd:keyword>_

**Comment by josusky** [comment]
Storing station identifiers as descriptiveKeywords is fine, and the XML is flexible enough so that both TSI and WSI can be stored. I also like your idea of adding link to OSCAR (e.g. "http://data.wmo.int/wigosid=0-20000-0-07145") but as a software developer I would love to see it stored as something else than "CharacterString" (with a custom format). Couldn't we use something more machine readable for example the (in)famous "gmx:Anchor" element?

**Comment by benjaminsaclier** [comment]
Yes fully agree with you Jan, but XML format remarks shall be address to the metadata group and I also support the idea to reduce "free text" in metadata and promote the usage of thesaurus
